src/cpp/rtps/security/SecurityManager.cpp

```diff
@@ -48,6 +47,7 @@
 
 #include <rtps/history/TopicPayloadPoolRegistry.hpp>
 #include <rtps/participant/RTPSParticipantImpl.h>
+#include <rtps/network/NetworkFactory.h>
```

Comment:
Please, sort alphabetically (VS Code has a command in the palette: sort lines ascending/descending)

Reply:
Nice to know